### ISSUE TITLE
How to change resolution browser?

### ISSUE THREAD

**Issue by henrytom1703** [alandmoore/wcgbrowser#20]
Hello,. How to change resolution browser?
Thanks

**Comment by alandmoore** [comment]
I don't understand your question; can you rephrase it or elaborate on it?

**Comment by henrytom1703** [comment]
Hello @alandmoore .
how to change a Screen Resolution of browser ?
Ex: Screen Resolution: 1366px x 768px at https://www.whoishostingthis.com/tools/user-agent/

Screen Resolution: | 1366px x 768px
-- | --




Screen Resolution: | 1366px x 768px
-- | --
